how-to-guides/sequential-pipelines/scripts/model_promotion.py

```diff
@@ -0,0 +1,41 @@
+import neptune.new as neptune
+import neptune.new.integrations.sklearn as npt_utils
+from neptune.new.exceptions import NeptuneException
+from neptune.new.types import File
+from utils import *
+
+model_name = "pickled_model"
+
+# (Neptune) Get latest model from training stage
+model_key = "PIPELINES"
+project_key = "SHOW"
+
+try:
+    model = neptune.init_model(
+        with_id=f"{project_key}-{model_key}",  # Your model ID here
+    )
+    model_versions_table = model.fetch_model_versions_table().to_pandas()
+    stagging_model_table = model_versions_table[model_versions_table["sys/stage"] == "staging"]
+    challenger_model_id = stagging_model_table["sys/id"].tolist()[0]
+    production_model_table = model_versions_table[model_versions_table["sys/stage"] == "production"]
+    champion_model_id = production_model_table["sys/id"].tolist()[0]
+
+except NeptuneException:
```

Comment:
```suggestion
except ModelNotFound:
```

Reply:
This exception is not documented.

Only NeptuneException, which is the parent and works for the majority of cases.

https://docs.neptune.ai/help/setting_up_error_handling_function/